explorer/app/components/Project/components/Publications/publications.tsx

```diff
@@ -0,0 +1,49 @@
+// Core dependencies
+import { Typography } from "@mui/material";
+import React from "react";
+
+// App dependencies
+import { ANCHOR_TARGET, Link } from "../../../Links/components/Link/link";
+import { SectionDetailsEmpty } from "../Section/components/SectionDetailsEmpty/sectionDetailsEmpty";
+import { Section } from "../Section/section";
+
+export interface Publication {
+  doi?: string;
```

Comment:
Would you mind please reviewing the `Publication` interface carefully - the API appears to always have `doi`, and `officialHcaPublication` (n.b. boolean and also null values observed), however the "updated project" ENTRIES model is optional `doi` and `officialHcaPublication`... Should I type the API separately?